spectrwm.c

```diff
@@ -8958,6 +9025,19 @@ setconfvalue(const char *selector, const char *value, int flags)
 		else
 			errx(1, "invalid bar_justify");
 		break;
+	case SWM_S_BAR_WS_LIST_SHOW:
+		if (strcmp(value, "active") == 0)
+			bar_ws_list_show = SWM_BAR_WS_LIST_ACTIVE;
+		else if (strcmp(value, "named") == 0)
+			bar_ws_list_show = SWM_BAR_WS_LIST_NAMED;
+		else if (strcmp(value, "all") == 0)
+			bar_ws_list_show = SWM_BAR_WS_LIST_ALL;
+		else
+			errx(1, "invalid bar_ws_list_show");
```

Comment:
should this blow up? or just show an error?

Reply:
Good question. I was basically treating this with similar severity as invalid values for other bar-related configs (e.g. justification).

If it would be preferable to adopt a warning in the bar - potentially in addition to a sensible default for the set of listed workspaces (e.g. `SWM_BAR_WS_LIST_ACTIVE`) - I'd certainly be open to that.

Reply:
I prefer a warning in the bar.  Then I can fix and M-q spectrwm quickly.

For the rest (I just tried it on OpenBSD-current), @swfrench's code works fine.  I like `active` best.  Urgency windows also prefix with `!`.  All useful features!

Reply:
Thanks for the follow-up! I'd be happy to swap out the errx for a warning in the bar. That said, it looks like @LordReg has put together an alternative impl of +L offering more fine-grained control over the included set of workspaces (rather than the three fixed flavors here).